translate/filters/checks.py

```diff
@@ -2155,10 +2161,31 @@ def accelerators(self, str1, str2):
         """Checks whether accelerators are consistent between the
         two strings.
 
-        For Mozilla we lower the severity to cosmetic.
+        For Mozilla we lower the severity to cosmetic, and for some languages
+        it also ensures accelerators are absent in the target string since some
+        languages do not use accelerators, for example Indic languages.
         """
+        if self.config.language_script:  # no-accelerators check
```

Comment:
The check isn't meant to not run if a script is set.  It is meant to not run when a script does not support accelerators.  The way that this is configured, any scripts defined for languages would cause accelerators not to work.  The idea of scripts is to be able to bulk configure operations.  I.e. accelerator check does not work for Devengari.  That way add a language code to Devengari and we automatically know that we don't need to run checks.

This seems like a fundamental oversite here ito implementation.